superset/Dockerfile

```diff
@@ -0,0 +1,14 @@
+# hadolint ignore=DL3006
+FROM apache/superset
```

Comment:
@jdangerx should we pin the base image version? If so, can we set up dependabot to keep the base image and the `requirements.txt` file up to date?